build-scripts/build-release.py

```diff
@@ -1398,7 +1398,7 @@ def main(argv=None) -> int:
     parser.add_argument('--vs-year', dest="vs_year", help="Visual Studio year")
     parser.add_argument('--android-api', dest="android_api", help="Android API version")
     parser.add_argument('--android-home', dest="android_home", default=os.environ.get("ANDROID_HOME"), help="Android Home folder")
-    parser.add_argument('--android-ndk-home', dest="android_ndk_home", default=os.environ.get("ANDROID_NDK_HOME"), help="Android NDK Home folder")
+    parser.add_argument('--android-ndk-home', dest="android_ndk_home", default=os.environ.get("ANDROID_NDK_LATEST_HOME"), help="Android NDK Home folder")
```

Comment:
Is this a standard Android NDK environment variable?
We want this script to not depend too much on GitHub Actions details.

Reply:
Honestly no but I'm not sure what else to do.  It is annoying to have this baked-in-to-the-image dependency.  I don't think GH will break this too readily as we would not be the first project to use it.